### PR TITLE
Refresh _ru/tour/tuples.md

### PR DESCRIPTION
**Pull request by artemkorsakov**
Refresh _ru/tour/tuples.md page and two links.

### CODE REVIEW

**Comment by artemkorsakov** [comment]
@kotobotov @DespiteDeath please review this PR

File: _ru/tour/tuples.md [hunk 1]
```diff
@@ -9,82 +9,134 @@ previous-page: traits
...
+val numPairs = List((2, 5), (3, -7), (20, 56))
+for (a, b) <- numPairs do
+  println(a * b)
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+## Кортежи и case-классы
```
Comment (DespiteDeath) [inline _ru/tour/tuples.md:133]: ## Кортежи и case-классы (классы-образцы) ?

maybe add an explanation in parentheses. In practice, the name "классы-образцы" seems to be rarely heard, more often they say "кейс-классы".

Comment (artemkorsakov) [reply _ru/tour/tuples.md]: Ok, we can use `"кейс-классы"`. Thanks for the review!

**Comment by artemkorsakov** [comment]
@sjrd @julienrf Could you please merge this approved PR?

**Review by sjrd — APPROVED** [review]

**Pull request merged**